sgx-isa/src/lib.rs

```diff
@@ -576,6 +580,24 @@ impl Report {
         // unwrap ok, `out` is the correct number of bytes
         Report::try_copy_from(&out.0).unwrap()
     }
+
+    #[cfg(all(feature = "sgxstd", target_env = "sgx"))]
+    pub fn verify<F, R>(&self, check_mac: F) -> R
```

Comment:
small suggestion:
Would it be possible to add documentation here to tell users that check_mac should prevent a timing attack?
Another possibility could be that we accept a mac call back and we equate macs of mac.

Reply:
Let's go with docs

Reply:
Just `// check_mac should prevent timing attacks` ?

Reply:
Well if you're documenting, might as well write the full docs for the function. Also, you should call out the comparison specifically as an area of concern.

PS. Doc comments are triple-slash

Reply:
Added comments